### PR TITLE
Code review exercise

### PR DESCRIPTION
**Pull request by jssol**
In this PR:

- [x] Copied files from the jssol/todo-list repo for a code review exercise

### CODE REVIEW

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hey @jssol , great job with the project. 
I can see you have implemented all the requirements of the project very well. I can see that you have implemented some features on your own like dark/light theme and i loved the way you implemented it. Also the structure of your functions is neat. However i would like to suggest some minor changes that you can implement to improve the code flow.

## Required Changes ♻️

- [x]  Check the comments under the review if any.
- [x]  Please consider to use a separete module file for bundling all the eventlistners together to maintain better code flow. This will also make your debugging easier as you scale up the project.
- [x] Consider creating a module folder inside src folder where you put all you module js files 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+To get the project up and running you need a browser.
+[The project live link](#Live-Demo-Link)
+
+### Install
+
+To get the project locally, run `git clone git@github.com:jssol/todo-list.git`
```
Comment (muneebulrehman) [inline README.md:35]: This command is not working

File: src/taskManager.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  addTask = (description) => {
+    if (description.trim()) {
+      const task = new Task(description.trim(), this.tasks);
+      this.tasks.push(task);
+    }
+  }
+
+  clearAll = () => {
+    this.tasks = [];
```
Comment (muneebulrehman) [inline src/taskManager.js:129]: Even though you are resetting the live array storing data, how are you updating local storage withing this function.

Comment (jssol) [inline src/taskManager.js:129]: The local storage has a separate function that updates it

**Pull request merged**